Prace_domowe/Praca_domowa2/Grupa1/Adam_Frej/PD2.ipynb

```diff
@@ -0,0 +1,2247 @@
+{
```

Comment:
<p>Dobrze, że poruszył Pan kwestię wyliczania błędu imputacji.</p><p>Błąd należałoby wyliczyć tylko dla wartości imputowanych, bo gdy wyliczamy dla całej ramki wynik jest zakłamany, ponieważ we wzorze jest wyliczana średnia, przez co liczność ma znaczenie.</p><p>To prawda, że wówczas porównywalibyśmy błąd wyliczony na różnych danych, ale to też jest ciekawe i pokaże jak w zależności od danych błąd się zmienia - dlatego eksperyment powtarzamy 10 razy.</p><p><u>Prośba o zmodyfikowanie sposobu wyliczania błędu.</u></p><p> </p><p>Szkoda, że nie wskazano wad poszczególnych metod encodingu w pierwszej części.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/111/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>